app/helpers/application_helper.rb

```diff
@@ -626,13 +626,15 @@ def discourse_color_scheme_stylesheets
   end
 
   def discourse_theme_color_meta_tags
-    result = +<<~HTML
```

Comment:
^ This removed the definition of `result`. This should fix the failures:

```
    result = +""
```

Reply:
Ah my bad, thanks :)